src/components/RSVPform.js

```diff
@@ -0,0 +1,175 @@
+import React, {useState} from "react";
+import {Form, Button, Col, FormControl, InputGroup} from "react-bootstrap";
+import {db} from "../firebase";
+
+const RSVPform = () => {
+  const [firstName, setFirstName] = useState("");
+  const [lastName, setLastName] = useState("");
+  const [rsvp, setRsvp] = useState("");
+  const [plusOne, setPlusOne] = useState(false);
+  const [plusFName, setPlusFName] = useState("");
+  const [plusLName, setPlusLName] = useState("");
+  const [allergies, setAllergies] = useState("");
+  const [guestEmail, setEmail] = useState("");
+
+  const addUser = () => {
+    const data = {
+      firstName: firstName,
+      lastName: lastName,
+      uid: new Date().getTime(),
+      rsvp: rsvp,
+      plusOne: plusOne,
+      plusFName: plusFName,
+      plusLName: plusLName,
+      allergies: allergies,
+      email: guestEmail,
+    };
+    db.collection("guests")
+      .doc(data.uid.toString())
+      .set(data)
+      .then(() => {
+        console.log("guest added!");
+      })
+      .catch((error) => {
+        console.log(error);
+      });
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    addUser();
+    setFirstName("");
```

Comment:
would it be worth creating a `clearFields` function?

Reply:
Done! Only thing is the plusOne checkbox is not cleared even though plusOne is set to false, thinking I may need to directly manipulate the DOM? Will do further research!